### PR TITLE
Volume Migration documentation added

### PR DESCRIPTION
**Pull request by walker2**
# Description

- Changed supported versions to align with 1.3
- Added Volume Migration page

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/265 |
| https://github.com/dell/csm/issues/266 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by alinamullagalieva — APPROVED** [review]

**Review by shanmydell — CHANGES_REQUESTED** [review]
The changes has to be maded in release notes, installation -> helm page as well

File: content/docs/csidriver/installation/helm/powermax.md [hunk 1]
```diff
@@ -232,6 +232,11 @@ CRDs should be configured during replication prepare stage with repctl as descri
 | sidecarProxyImage | Image for csm-authorization-sidecar. | No | " " |
 | proxyHost | Hostname of the csm-authorization server. | No | Empty |
 | skipCertificateValidation | A boolean that enables/disables certificate validation of the csm-authorization server. | No | true |
+| **migration** | [Migration](../../../../replication/migrating-volumes) is an optional feature to enable migration between storage classes | - | - |
+| enabled                  | A boolean that enables/disables replication feature. |  No      |   false   |
```
Comment (nidtara) [inline content/docs/csidriver/installation/helm/powermax.md:236]: This should be "migration feature"

File: content/docs/replication/migrating-volumes.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+```shell
+NAME                                   CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS      CLAIM                       STORAGECLASS                REASON   AGE
+test-pv                                1Gi        RWO            Retain           Bound       default/test-pvc            powermax                             5m
+test-pv-to-powermax-replication        1Gi        RWO            Retain           Available                               powermax-replication                 10s
+
+```
+
+When Volume Migration is finished source PV will be updated with EVENT that denotes that. 
+
+Newly created PV (`test-pv-to-powermax-replication` in our example) is available for consumption via static provisioning by any PVC that will request it.
```
Comment (nidtara) [inline content/docs/replication/migrating-volumes.md:52]: Post migration, will we support other use-cases like volume expansion, snapshot creation?

File: content/docs/csidriver/installation/helm/powermax.md (hunk 1 above)
Comment (walker2) [reply content/docs/csidriver/installation/helm/powermax.md:236]: Yes, thanks will change it

**Review by maxan98 — APPROVED** [review]

**Review by boyamurthy — DISMISSED** [review]
LGTM.

**Review by shanmydell — APPROVED** [review]

**Review by boyamurthy — APPROVED** [review]

**Pull request merged**